src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static ArrayList<Task> taskList = new ArrayList<>();
+
+    public static void main(String[] args) throws DukeException {
+        System.out.println("Hi there, I'm TARS!\nWhat can I do for you?");
+        ReadWriteFile.readTaskList(taskList);
+//        for (Task t : taskList) {
+//            System.out.println(t);
+//        }
+        Scanner scanner = new Scanner(System.in);
+        boolean stop = false;
+
+        while (!stop) {
+            String[] line = scanner.nextLine().split(" ", 2);
+            String operation = line[0];
+            switch (operation) {
+                case "bye":
+                    stop = true;
+                    System.out.println("Bye bye. See you again soon!");
+                    break;
+
```

Comment:
According to the Java coding standard guide, under the Layout section, I think it shows a sample formatting of a switch statement and in it, there is no blank newline after the `break`. Should there be a blank newline after the `break` in the following code?

Reply:
I'm not too sure about this either, but I should remove it to keep my code clean since it doesn't improve readability also